README.md

```diff
@@ -19,118 +19,8 @@ This response is then used to configure MediaWiki.
-This is run as a step in `sync.sh`
-
-### sync.sh
-
-This script will resync the WHOLE git repo.
-
-This is made up of a series of steps:
-
-- `01-clear`: Remove all MediaWiki code
-- `pacman`: Retrieive and build MediaWiki code
-- `03-less-files`: Remove not needed things from MediaWiki code
-- `04-docker-compose`: Perform a composer install
-- `05-docker-entrypoint-overrides`: Add the WBStack shims to MediaWiki entrypoints
-
-## MediaWiki Loading
-
-MediaWiki loads this code in 2 ways.
-
-Firstly through entry point shims:
-
-- **(index|load|api|rest).php** - MediaWiki entry points
-  - src/Shim/*.php - These files are loaded at the start of the MediaWiki entry points
-    - src/loadShim.php
-      - src/Info/WBStackInfo.php - Main code for fetching things from the platform API
-      - src/Logging/WikWikiSpi.php
-      - src/Logging/WikWikiLogger.php
-
-And secondly via LocalSettings.php
-
-- **LocalSettings.php** - The actual MediaWiki settings file
-  - src/Settings/LocalSettings.php - Is loaded from the MediaWiki LocalSettings.php (where it normally would be)
-    - src/loadAll.php
-    - src/Settings/Cache.php
-    - src/loadInternal.php - Only loaded for the INTERNAL flavour of the app.
-      - src/Internal/*
-    - src/Settings/Hooks.php
-
-## Secondary setup
-
-### ElasticSearch index configuration
-
-In order to enable elasticsearch the `UpdateSearchIndexConfig.php` needs to be executed for that wiki.
-On wiki creation through the API this is done by the `ApiWbStackElasticSearchInit` job.
-
-## Development Environment
-
-There is a [docker-compose file](../docker-compose.yml) in the root directory that allows for serving multiple development sites locally.
-
-These are currently not using the real api but rather gets their settings from the static json files included in the data folder.
-
-The fake api is served by the [server.php](test/server.php) script and reads the corresponding [subdomain](data/WikiInfo-site1.json) from each request.
-
-
-### Start the dev environment
-
-```sh
-docker-compose up
-```
-
-Wait until both sites are accessible:
-
- - http://site1.localhost:8001/wiki/Main_Page
- - http://site2.localhost:8001/wiki/Main_Page
-
- You may need to add an entry to your `hosts` file:
-
- ```
- 127.0.0.1 site1.localhost site2.localhost
- ```
-
- Once the sites are accessible you can perform secondary setup (_The request takes a while to execute_):
-
- ```sh
-curl -l -X POST "http://site1.localhost:8001/w/api.php?action=wbstackElasticSearchInit&format=json"
-curl -l -X POST "http://site2.localhost:8001/w/api.php?action=wbstackElasticSearchInit&format=json"
-```
-
-### Debugging Elastic
-
-General overview of the cluster
-
-```
-http://localhost:9200/_stats
-```
-
-Entries in the content index (Items, Lexemes) for `site1.localhost` can be found by going to the following url
-
-```
-http://localhost:9200/site1.localhost_content_first/_search
-```
+- [Environment variables](./docs/environment-vars.md)
```

Comment:
Maybe we should add some short explanations to these links to make it clear what why are linking to